templates/openstack/consulting.html

```diff
@@ -317,31 +317,32 @@ <h3 class="p-matrix__title">Full-stack support</h3>
           <div class="p-matrix__content">
             <h3 class="p-matrix__title">10 years of security</h3>
             <p class="p-matrix__desc">
-              Not every organisation is willing or able to upgrade their cloud on a regular basis. With Charmed OpenStack you stay secure for up to 10 years under Canonical’s <a href="/security/esm">Expanded Security Maintenance (ESM)</a> programme with no need to upgrade.
+              Not every organisation is willing or able to upgrade their cloud on a regular basis. With Charmed OpenStack you stay secure for up to 10 years under Canonical's <a href="/security/esm">Expanded Security Maintenance (ESM)</a> program with no need to upgrade.
```

Comment:
```suggestion
              Not every organization is willing or able to upgrade their cloud on a regular basis. With Charmed OpenStack you stay secure for up to 10 years under Canonical's <a href="/security/esm">Expanded Security Maintenance (ESM)</a> program with no need to upgrade.
```
BE->AE. This wasn't in the copy doc, I've added it as a suggestion